packages/contentstack-import/src/utils/entries-helper.ts

```diff
@@ -534,11 +550,37 @@ export const restoreJsonRteEntryRefs = (
         }
         break;
       }
+      case 'text': {
+        if (entry[element.uid] && element.field_metadata.rich_text_type) {
+          entry[element.uid] = sourceStackEntry[element.uid];
+          const matches: string[] = [];
+          Object.keys(uidMapper).forEach((uid) => {
+            if (sourceStackEntry[element.uid].indexOf(uid) !== -1) {
+              matches.push(uid);
+            }
+          });
```

Comment:
@cs-raj here can we use filter? For example:-

const matches = Object.keys(uidMapper).filter((uid) =>  if (sourceStackEntry[element.uid].indexOf(uid) !== -1) return uid;)

Reply:
Added